lib/graphql/subscriptions.rb

```diff
@@ -228,7 +228,10 @@ def normalize_arguments(event_name, arg_owner, args)
         # match query arguments.
         arg_owner.arguments.each do |name, arg_defn|
           if arg_defn.default_value? && !normalized_args.key?(arg_defn.name)
-            normalized_args[arg_defn.name] = arg_defn.default_value
+            default_value = arg_defn.default_value
+            # We don't have an underlying "object" here, so it can't call methods.
+            # This is broken.
+            normalized_args[arg_defn.name] = arg_defn.prepare_value(nil, default_value, context: GraphQL::Query::NullContext)
```

Comment:
I thought, "maybe instead of adding this, I should change the `default_value:` to the already-prepared value (that is, `"ONE"` to `"one"` in this code). But, that failed:

```
GraphQL::Schema::InvalidTypeError: Subscription is invalid: field "eventSubscription" argument "payloadType" default value "one" is not valid for type PayloadType (Can't resolve enum PayloadType for "one")
```